tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/BuilderUtil.scala

```diff
@@ -41,7 +41,7 @@ object BuilderUtil {
       } yield seqEx :+ argEx
     }
 
-  /** Lifts a binary raw method to a BuilderWrapper method */
+  /** Lifts a binary unsafe method to a TBuilderInstruction method */
```

Comment:
Since this name is imported we could link it to make sure IDEA picks it up on refactoring.

```suggestion
  /** Lifts a binary unsafe method to a [[TBuilderInstruction]] method */
```